docs/components/safety.md

```
@@ -121,6 +121,10 @@ blocking if you cancel an unsafe run.
 | `--export-safety-data [FILE]` | Save safety data as JSON, optionally to a custom filename. |
 | `--retry-with-safety-on-interrupt` | Re-run once with system blocking after Ctrl-C. |
 | `--no-network` / `--allow-host HOST` | Restrict outbound HTTP targets. |
+| `--runtime-probe` | Enable mcpfz-probe runtime monitoring for stdio server behavior. |
+| `--runtime-probe-backend ebpf|fake|auto` | Select the sidecar backend. |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)